python/fastquant/strategies.py

```diff
@@ -516,6 +517,97 @@ def sell_signal(self):
         return self.buy_and_hold_sell
 
 
+class Sentiment(bt.Indicator):
```

Comment:
@rafmacalaba  I think it makes sense to put this in a new `indicators.py` module, so that `strategies.py` is purely for strategy definitions :smile: